Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingList/OfferingButton.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  OfferingButton.swift
+//  PaywallsTester
+//
+//  Created by James Borthwick on 2024-04-30.
+//
+
+import SwiftUI
+import RevenueCat
+
+struct OfferingButton: View {
+
+    var body: some View {
+        Button {
+            showPaywall()
+        } label: {
+            label()
+        }
+        #if !os(watchOS)
+        .contextMenu {
+            contextMenuItems()
+        }
+        #endif
+    }
+
+    init(offeringPaywall: OfferingPaywall,
+         viewModel: OfferingsPaywallsViewModel,
+         selectedItemID: Binding<String?>) {
+        self.responseOffering = offeringPaywall.offering
+        self.rcOffering = offeringPaywall.rcOffering
+        self.viewModel = viewModel
+        self._selectedItemID = selectedItemID
+    }
+
+    private let responseOffering: OfferingsResponse.Offering
+    private let rcOffering: Offering
+    private let viewModel: OfferingsPaywallsViewModel
+    @Binding private var selectedItemID: String?
+}
+
+private extension OfferingButton {
+    private func showPaywall(mode: PaywallViewMode = .default) {
+        Task { @MainActor in
+            await viewModel.getAndShowPaywallForID(id: responseOffering.id, mode: mode)
+            selectedItemID = responseOffering.id
+        }
+    }
+}
+
+private extension OfferingButton {
+
+    @ViewBuilder
+    private func label() -> some View {
+        let templateName = rcOffering.paywall?.templateName
+        let paywallTitle = rcOffering.paywall?.localizedConfiguration.title
+        let decorator = viewModel.hasMultipleOfferingsWithPaywalls && self.selectedItemID == responseOffering.id ? "▶ " : ""
+        HStack {
+            VStack(alignment:.leading, spacing: 5) {
+                Text(decorator + responseOffering.displayName)
+                    .font(.headline)
+                if let title = paywallTitle, let name = templateName {
+                    let text = viewModel.hasMultipleTemplates ? "Style \(name): \(title)" : title
+                    Text(text)
+                        .font(.footnote)
+                        .foregroundColor(.secondary)
```

Comment:
super unrelated, but... we should maybe think about updating the tint colors to RevenueCat Red rather than the orange that we added I can't remember why

Reply:
I had thought the same thing! Works in both light and dark mode. Done!
![image](https://github.com/RevenueCat/purchases-ios/assets/109382862/b6501758-4566-4233-9e58-13cb0ee3bf61)
![image](https://github.com/RevenueCat/purchases-ios/assets/109382862/7d6f5f8d-ea8a-469e-96b6-2f41a809ef2c)

Reply:
nice, love it!!